code/modules/clothing/spacesuits/rig/rig_attackby.dm

```diff
@@ -37,6 +37,10 @@
 	var/tool_type = I.get_tool_type(user, usable_qualities)
 	switch(tool_type)
 		if(QUALITY_SCREW_DRIVING)
+			if (is_worn())
```

Comment:
Any reason for this change? It is inconsistent and only applies to modules and ignores air tanks.

Reply:
it was moved from a few lines down

Reply:
fixing